example/android/gradlew.bat

```diff
@@ -60,11 +59,6 @@ set _SKIP=2
 if "x%~1" == "x" goto execute
 
 set CMD_LINE_ARGS=%*
-goto execute
-
-:4NT_args
-@rem Get arguments from the 4NT Shell from JP Software
-set CMD_LINE_ARGS=%$

```

Comment:
Why the support for `Take Command Console` removed here?
AFAIK, it's not deprecated. As I saw its latest stable releases is 20.11 on March 10, 2017, 6 months ago. And it's supporting Windows 10, the latest Windows release.

Reply:
Please see my comment above, this file is also auto-generated by the `wrapper` task and thus comes straight from Gradle.